### PR TITLE
EIP-7594: Add asserts for public functions

### PR DESCRIPTION
**Pull request by jtraglia**
I'm working on the tests generator for this and noticed that these functions weren't raising an exception for invalid blobs. This is because we forgot to add length asserts in the public functions.

### CODE REVIEW

**Review by kevaundray — APPROVED** [review]

**Review by kevaundray — APPROVED** [review]

File: specs/_features/eip7594/polynomial-commitments-sampling.md [hunk 1]
```diff
@@ -463,6 +473,14 @@ def verify_cell_proof_batch(row_commitments_bytes: Sequence[Bytes48],
...
     """
     assert len(cells_bytes) == len(proofs_bytes) == len(row_indices) == len(column_indices)
+    for commitment_bytes in row_commitments_bytes:
+        assert len(commitment_bytes) == BYTES_PER_COMMITMENT
+    for cell_bytes in cells_bytes:
+        assert len(cell_bytes) == FIELD_ELEMENTS_PER_CELL
+        for field_bytes in cell_bytes:
+            assert len(field_bytes) == BYTES_PER_FIELD_ELEMENT
+    for proof_bytes in proofs_bytes:
+        assert len(proof_bytes) == BYTES_PER_PROOF
```
Comment (asn-d6) [inline specs/_features/eip7594/polynomial-commitments-sampling.md:484]: Do we also want to check that row_indices and column_indices are within the bounds here?

Comment (jtraglia) [reply specs/_features/eip7594/polynomial-commitments-sampling.md:484]: Yes, that's a good idea. Will add that.

**Comment by jtraglia** [comment]
When merging this, please squash.

**Review by asn-d6 — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**